api.raml

```diff
@@ -189,16 +189,20 @@ types:
           description: unique identifier provided by the UI client
           type: string
         ReturnTo:
-          description: URL to return to after login
+          description: URL to return to after login or invite
+          type: string
+        ReturnToOnError?:
+          description: URL to return to in case of expired invite
           type: string
         invite?:
           description: New user invite code
           type: string
       responses:
         302:
           description: >
-            Redirection to `ReturnTo` with `state`, `token_type`, `expires_utc` and
-            `access_token` appended as query string parameters.
+            On successful login or invite, a redirection to `ReturnTo` with `state`, `token_type`,
+            `expires_utc` and `access_token` appended as query string parameters. On error, a
+            redirection to `ReturnToOnError` with no query string parameters.
```

Comment:
This makes it sound like the ReturnToOnError applies both to resets and to invites, but based on the code it looks like it only applies to invites. I'm not sure which was intended.

Reply:
I clarified the description.